example/user/user.pb.gorm.go

```diff
@@ -169,7 +169,7 @@ func (m *User) ToORM(ctx context.Context) (UserORM, error) {
 	if m.ExternalUuid != nil {
 		if v, err := resource1.Decode(nil, m.ExternalUuid); err != nil {
 			return to, err
-		} else {
+		} else if v != nil {
 			vv := v.(string)
```

Comment:
The `nil` check is good, but should we do a check on this type assertion, too? Or are we okay with, panic-ing if the DB type is wrong (what would cause this assert to fail, I believe) since that's a big error, or should we do something more "graceful"?